cfme/utils/appliance/__init__.py

```diff
@@ -1012,17 +1012,25 @@ def ssh_client(self):
             }
         if self.is_dev:
             connect_kwargs.update({'is_dev': True})
-        ssh_client = ssh.SSHClient(**connect_kwargs)
-        try:
-            ssh_client.get_transport().is_active()
-            logger.info('default appliance ssh credentials are valid')
-        except Exception as e:
-            if self.is_dev:
-                raise Exception('SSH access on a dev alliance, (unsupported)')
-            logger.error(e)
-            logger.error('default appliance ssh credentials failed, trying establish ssh connection'
-                         ' using ssh private key')
-            ssh_client = self.ssh_client_with_privatekey()
+
+        def create_ssh_connection():
+            ssh_client = ssh.SSHClient(**connect_kwargs)
+            try:
+                ssh_client.get_transport().is_active()
+                logger.info('default appliance ssh credentials are valid')
+            except Exception as e:
+                if self.is_dev:
+                    raise Exception('SSH access on a dev alliance, (unsupported)')
+                logger.error(e)
```

Comment:
This can be cleanly logged with `logger.exception('default appliance ssh credentials failed, establish ssh connection using ssh private key')`.  Can clean that up while we're making changes here anyway :)